ansible/roles/druid-ingestion/templates/raw_sl_observation_status

```diff
@@ -0,0 +1,201 @@
+{
+  "type": "index",
+  "spec": {
+    "ioConfig": {
+      "type": "index",
+      "firehose": {
+        "type": "static-azure-blobstore",
+        "blobs": [
+          {
+            "container": "{{container_name}}",
```

Comment:
Do we have container_name variable specified in defaults?

Reply:
Added this variable. Please check